migrate/v0_15/migrate.go

```diff
@@ -141,6 +148,96 @@ func MigrateAccounts(genesisState auth.GenesisState, genesisTime time.Time) auth
 	return migratedGenesisState
 }
 
+// DistributeSwpTokens sets the initial distribution of swp tokens according to the proposed token supply schedule in prop 59
+func DistributeSwpTokens(genesisState auth.GenesisState) auth.GenesisState {
+	accounts := make([]authexported.GenesisAccount, len(genesisState.Accounts))
+	migratedGenesisState := auth.NewGenesisState(genesisState.Params, accounts)
+	// add SWP incentives (LP, Kava stakers) to kavadist module account
+	kavaDistAddr := supply.NewModuleAddress(kavadist.KavaDistMacc)
+	for i, acc := range genesisState.Accounts {
+		if acc.GetAddress().Equals(kavaDistAddr) {
+			err := acc.SetCoins(acc.GetCoins().Add(sdk.NewCoin("swp", sdk.NewInt(137500000e6))))
```

Comment:
```suggestion
			err := acc.SetCoins(acc.GetCoins().Add(sdk.NewCoin("swp", sdk.NewInt(kavaDistAddrSwpAmount))))
```

nit: making the SWP token amounts constants defined at the top of the file or function would be clearer.

Reply:
Could also then use the values to test individual addresses' token balances